python/ucxx/ucxx/_lib_async/application_context.py

```diff
@@ -46,7 +46,7 @@ def __init__(
         progress_mode=None,
         enable_delayed_submission=None,
         enable_python_future=None,
-        exchange_peer_info_timeout=10.0,
+        connect_timeout=10.0,
```

Comment:
```suggestion
        connect_timeout=None,
```

Reply:
Let's switch to `None`, like in UCX-Py, this allows us to simplify the conditional.